pkg/cmd/bind.go

```diff
@@ -206,6 +189,14 @@ func (o *bindCmdOptions) run(args []string) error {
 	return nil
 }
 
+func showOutput(cmd *cobra.Command, binding *v1alpha1.KameletBinding, outputFormat string) error {
+	printer := printers.NewTypeSetter(scheme.Scheme)
```

Comment:
That's the default scheme. My understanding is that the scheme initialised for the command client should be passed.

Reply:
Yes, I thought it was not available in the `cmd`, but I manage to find it moving the output logic after creating the `client`. Fixed now, thanks!